src/Form/InvoiceEntryWorklogFilterType.php

```diff
@@ -11,8 +12,14 @@
 
 class InvoiceEntryWorklogFilterType extends AbstractType
 {
+    public function __construct(
+        private readonly InvoiceRepository $invoiceRepository,
+    ) {
+    }
+
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
+        $invoice = $this->invoiceRepository->find($options['invoiceId'] ?? null);
```

Comment:
Change the options['invoiceId'] to options['periodFrom'] , options['periodTo']  with correct data type.
Then you avoid trying to retrieve the invoice inside the filter type